test/models/recipe_food_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class RecipeFoodTest < ActiveSupport::TestCase
+  # test "the truth" do
+  #   assert true
+  # end
+end
```

Comment:
- [ Required ] Please consider adding `Unit` and `Integration` tests to your project, currently, when I run the tests in your project it shows no tests have been added. Please see the screenshot below to see what happened when I ran 
```sh 
rspec spec
```
## Screenshot
![Screenshot from 2023-02-18 15-26-13](https://user-images.githubusercontent.com/16392046/219865654-274e6d4d-ba79-43d3-99a3-9df7ed6fa161.png)

## Project requirements
![Screenshot from 2023-02-18 15-40-23](https://user-images.githubusercontent.com/16392046/219866383-13adee5f-7701-486f-87db-7af51eebba5b.png)

Reply:
Will do 👍 . Thank you